browser/brave_local_state_prefs.cc

```diff
@@ -60,6 +62,11 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
 
   brave_shields::RegisterShieldsP3APrefs(registry);
 #if !defined(OS_ANDROID)
+  RegisterPrefsForMuonMigration(registry);
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT) || BUILDFLAG(BUNDLE_WIDEVINE_CDM)
```

Comment:
I think this should move out of the `!defined(OS_ANDROID)` block since it has its own guards

Reply:
Fixed.